jvm/jfr-annotations/src/main/java/org/quickperf/jvm/jfr/JFREventsLoader.java

```diff
@@ -42,17 +43,20 @@ private IItemCollection load(String jfrFilePath) {
         try {
             File jfrFile = new File(jfrFilePath);
             IItemCollection jfrEvents = JfrLoaderToolkit.loadEvents(jfrFile);
-            String pointingRight = "\uD83D\uDC49";
-            System.out.println("[QUICK PERF] JVM was profiled with JDK Flight Recorder (JFR)."
-                              + System.lineSeparator()
-                              + "The recording file is available here: "
-                              + jfrFilePath
-                              + System.lineSeparator()
-                              + "You can open it with JDK Mission Control (JMC)."
-                              + System.lineSeparator()
-                              + "Where to find JDK Mission Control? " + pointingRight + " https://tinyurl.com/jdkmc"
-                               )
-                              ;
+            boolean tlabAndOutsideTlabEventsDisabled = jfrEvents.apply(JdkFilters.ALLOC_OUTSIDE_TLAB).hasItems()
```

Comment:
The JVM is profiled, but some memory metrics are not shown by default with the JDK 16. So, please don't modify the JFREventsLoader.java  file.